include/render/ShaderUniforms.h

```diff
@@ -0,0 +1,20 @@
+﻿
+#include "common/common.h"
+
+
+class ShaderUniforms {
+
+public:
+
+	static std::vector<std::string> shader_uniforms;
+
+};
+
+
+
+
+
+
+
+
+
```

Comment:
Lots of extra lines that can be deleted

```suggestion
#include "common/common.h"

class ShaderUniforms {
public:
	static std::vector<std::string> shader_uniforms;
};
```
```